src/azure-client.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from "fs";
+
+import FormData from "form-data";
+
+import { getApiClient } from "./gh-api-client";
+
+export interface Policy {
+  upload_url: string;
+  header: Map<string, string>;
+  form: Map<string, string>;
+}
+
+export async function uploadArtifact(policy: Policy, artifactZipPath: string) {
+  const data = new FormData();
+  for (const [key, value] of policy.form.entries()) {
```

Comment:
If you change the type of `policy.form` then this will need to change as follows:
```suggestion
  for (const [key, value] of Object.entries(policy.form)) {
```